tests/foreman/cli/test_oscap.py

```diff
@@ -60,14 +72,42 @@ def create_test_user_viewer_role(cls):
         return cls.login, cls.password
 
     @classmethod
+    def fetch_scap_and_profile_id(cls, scap_name, scap_profile):
+        """Extracts the scap ID and scap profile id
+
+        :param scap_name: Scap title
+        :param scap_profile: Scap profile you want to select
+
+        :returns: scap_id and scap_profile_id
+        """
+        default_content = Scapcontent.info({'title': scap_name})
+        scap_id = default_content['id']
+        scap_profile_id = (
```

Comment:
- what is the type scap_profile_id
if it's a list of of ids it must be scap_profile_ids

- also why we putting that between the parentheses "()"
```python
scap_profile_ids =  [
    profile['id']
    for profile in default_content['scap-content-profiles']
    if scap_profile in scap['title']
]
```
the above should be sificient 

what do you think?